### PR TITLE
Update settings.md

### PR DESCRIPTION
**Pull request by seneca**
Edits.

### CODE REVIEW

File: packages/nuejs.org/docs/settings.md [hunk 1]
```diff
@@ -37,46 +37,46 @@ links:
...
 ### direction
-The value of `<html direction="{ direction }">` attribute. The default is "ltr" (left to right)
+The value of `<html direction="{ direction }">` attribute. The default is "ltr" (left to right).
 
 ### favicon
 Relative path to your favicon that overrides the "favicon.ico" on the browser tab.
 
 ### generator
-The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered for the production pages.
+The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered on the production pages.
```
Comment (nobkd) [inline packages/nuejs.org/docs/settings.md]:
```suggestion
The generator meta tag. The default is "Nue (nuejs.org)" and is only rendered in production mode.
```
(I think `for` fits best, but I understand, that `on` emphasizes, that it's in the page's html)

Or maybe just `in production (mode)`?

**Review by nobkd — CHANGES_REQUESTED** [review]

File: packages/nuejs.org/docs/settings.md (hunk 1 above)
Comment (seneca) [reply packages/nuejs.org/docs/settings.md]: I think in production (mode) seems the best choice.

Comment (nobkd) [reply packages/nuejs.org/docs/settings.md]: Okay. 
Do you like `in production mode.` or `in production.` better?

Comment (seneca) [reply packages/nuejs.org/docs/settings.md]: I think in production mode makes it clearer what it is.

**Review by nobkd — APPROVED** [review]

**Comment by nobkd** [comment]
No significant changes for this file in merge commit.

**Pull request merged**